notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -22,7 +22,7 @@
   },
```

Comment:
<p>Here, you should try reading from ceph. Make a <code>.env</code>  file and use the credentials mentioned here - https://github.com/operate-first/support/issues/165#issuecomment-810337239 and you would be able to get the dataset from ceph instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>